app/src/main/res/layout/activity_login.xml

```diff
@@ -65,6 +66,7 @@
         <android.support.v7.widget.AppCompatEditText
             android:layout_width="match_parent"
             android:layout_height="49dp"
+            android:id="@+id/pass_editText"
```

Comment:
Can you please name this ID as "etPassword" to follow the other ids format in the project?